.gitignore

```diff
@@ -195,3 +195,4 @@ FakesAssemblies/
 # Visual Studio 6 workspace options file
 *.opt
 *.orig
+Collections/Collections.csproj
```

Comment:
csproj should NOT be ignored by git in any case!!!
You just will not be able to update your project with new files, or see it in newly cloned code.